src/index.html

```diff
@@ -5,9 +5,57 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700;800&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./utils/main.scss">
   </head>
+
   <body>
-    <h1>Product cards</h1>
+
+    <div class="card" data-qa="card">
+      <div class="card__items">
+        <img
+          class="card__icon"
+          src="./images/imac.jpeg"
+          alt="Apple tv"
+        >
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+      </div>
+
+      <p class="card__product">
+        Product code: 195434
+      </p>
+
+      <div class="card__star">
```

Comment:
it's no only a card star. So it's better to give more meaningful name such as card rating
```suggestion
      <div class="card__rating">
```